refocus-bdk-server.js

```diff
@@ -361,6 +362,8 @@ module.exports = (config) => {
     });
 
     socket.on('connect', () => {
+      socket.emit('auth', token);
```

Comment:
Nice, is this the new way to authenticate? We were thinking we would need something like this at the minute, but maybe we don't now: https://github.com/salesforce/refocus-bdk/pull/23/files#diff-57a3a2eee7975d9623a06fabdb1bd15aR302

Reply:
Yes, this is the new auth method that will be used with the new namespace format.
The first iteration of the separate realtime app will still use the format with the token in the query param, so you will still need to support that as well.